lib/helpers/wraith_config_helper.rb

```diff
@@ -0,0 +1,47 @@
+class WraithConfigHelper
+  attr_accessor :name, :paths
+
+  TEMPLATE_PATH = "test/wraith/config.yaml".freeze
+  OUTPUT_PATH = 'test/wraith/wip-config-%{suffix}.yaml'.freeze
+
+  def initialize(name, paths)
+    @name = name
+    @paths = paths
+  end
+
+  def create_config(extra_config = {})
+    config = load_template
+    config["paths"] = build_paths
+    config.merge!(extra_config) if extra_config.present?
+
+    write_config(config)
+  end
+
+private
+
+  def load_template
+    YAML::load(File.open(TEMPLATE_PATH))
+  end
+
+  def build_paths
+    config_paths = {}
+
+    if @paths.is_a? Hash
+      @paths.keys.each do |key|
+        @paths.fetch(key).each_with_index do |path, index|
+          config_paths["#{key}_#{index + 1}"] = path
+        end
+      end
+    else
+      @paths.each_with_index { |path, index| config_paths["#{name}_#{index + 1}"] = path }
```

Comment:
Minor: `config_paths["#{name}_#{index + 1}"] = path` is repeated, could be DRY'd up